dcpy/library/archive.py

```diff
@@ -24,6 +24,7 @@ def __call__(
         clean: bool = False,
         latest: bool = False,
         name: str | None = None,
+        path_override: str | None = None,
```

Comment:
do you want to add this to the function docstring below? And is it worth specifying that this is a `source_path_override`, to distinguish from the `path` arg above?

Reply:
and just to check that I understand, `path` is where the config file lives, and this `path_override` sets the location for where the dataset is?

Reply:
Yes to all of the above